KomaMRIBase/src/datatypes/Phantom.jl

```diff
@@ -612,5 +535,59 @@ function check_phantom_arguments(nd, ss, us)
             ssy = ss[2]
         end
     end
+    
     return ssx, ssy, ssz, usx, usy, usz
 end
+
+"""
+    ρ, T1, T2, T2s, Δw = default_brain_tissue_properties(labels, tissue_properties = nothing)
+
+This function returns the default brain tissue properties using a labels identifier Matrix
+# Arguments
+- `labels` : (`::Matrix`) the labels identifier matrix of the phantom
+- `tissue_properties` : (`::Dict`, `=Dict()`) phantom tissue properties in ms and Hz considering the available tissues
+
+# Returns
+- `ρ, T1, T2, T2s, Δw`: (`::Matrix`) matrices of the same size of labels with the tissues properties information
+
+# Examples
+```julia-repl
+julia> ρ, T1, T2, T2s, Δw = default_brain_tissue_properties(labels, tissue_properties)
+
+julia> ρ, T1, T2, T2s, Δw = default_brain_tissue_properties(labels)
+```
+"""
+function default_brain_tissue_properties(labels, tissue_properties = Dict())
+    # Load default tissue properties
+    default_properties = Dict(
+        # ρ, T1, T2, T2*, Δw
+        "CSF"           => [1,      2.569,  0.329,  0.058,  0],
+        "GM"            => [0.86,   0.833,  0.083,  0.069,  0],
+        "WM"            => [0.77,   0.500,  0.070,  0.061,  0],
+        "FAT1"          => [1,      0.350,  0.070,  0.058,  -220*2π], #-220 Hz
+        "MUSCLE"        => [1,      0.900,  0.047,  0.030,  0],
+        "SKIN/MUSCLE"   => [1,      0.569,  0.329,  0.058,  0],
+        "SKULL"         => [0,      0,      0,      0,      0],
+        "VESSELS"       => [0,      0,      0,      0,      0],
+        "FAT2"          => [0.77,   0.500,  0.070,  0.061,  -220*2π], #-220 Hz
+        "DURA"          => [1,      2.569,  0.329,  0.058,  0],
+        "MARROW"        => [0.77,   0.500,  0.070,  0.061,  0])
+    
+    tissue_properties = merge(default_properties, tissue_properties)
+    props = ["ρ", "T1", "T2", "T2s", "Δw"]
+    Nproperties = length(props) 
+    tissue_labels = [23, 46, 70, 93, 116, 139, 162, 185, 209, 232, 255]
+    tissue_texts = ["CSF", "GM", "WM", "FAT1", "MUSCLE", "SKIN/MUSCLE", "SKULL", "VESSELS", "FAT2", "DURA", "MARROW"]
```

Comment:
You are defining this twice, use the keys of the `default_properties` dictionary, like in my example.

`data_properties[i, ntuple(j -> 1:dims[j+1], length(dims)-1)...]`  this is not very elegant, just do `data_properties[i, :, :, :]`, no need to define `dims` as the data is always three dimensional.

Reply:
Yeah, well, the problem is that the keys of the dictionary come in a different order, i.e., `keys(tissue_properties)` is not consistent with the creation order. 

Also, the problem with doing `data_properties[i, :, :, :]` is the inconsistent shape that `labels` has between 2D and 3D phantoms. The original code produces `labels` with 2 or 3 dimensions depending on what is being asked (`brain_phantom2D` or `brain_phantom3D` - which kind of makes sense but it also makes sense to introduce a singleton dimension on the `brain_phantom2D`'s labels to then generalize the code to 3D). In any case, this inconsistency made `data_properties[i, ntuple(j -> 1:dims[j+1], length(dims)-1)...]` required. 

Nevertheless, to don't use those lines, I modified the code.